lib/ansible/module_utils/profitbricks.py

```diff
@@ -0,0 +1,105 @@
+# This code is part of Ansible, but is an independent component.
```

Comment:
Please moved to `module_utils/cloud/profitbricks/profitbricks.py` you will need to create `__init__.py` files in any new directories.

Please also add this new directory to https://github.com/ansible/ansible/blob/devel/.github/BOTMETA.yml

Reply:
@gundalow , just to make sure before I commit the changes, the parent dir `module_utils/cloud` does not exist either. `profitbricks` is the first util module that is to be residing under `module_utils/cloud` , right?

Reply:
Correct